### ISSUE TITLE
download counts from "< 1%" are omitted when hidden

### ISSUE THREAD

**Issue by ljharb** [NullVoxPopuli/package-majors#70]
iow, if there's a ton of minors with < 1% downloads, and it's set to "majors" and "hide < 1%", then the counts will be off.

See webpack for an example.

**Comment by NullVoxPopuli** [comment]
thank you for the issue report!

**Comment by ljharb** [comment]
yes - https://majors.nullvoxpopuli.com/q?minors=false&old=false&packages=webpack - v5 has 90.2%, but if you "show < 1%" it has 86.2%. The percentages and counts should always be of the full total, regardless of what's displayed.

**Comment by ljharb** [comment]
thanks, looks great!
